.github/workflows/pulp_images.yml

```diff
@@ -271,14 +271,14 @@ jobs:
           sed -i "s/pulp-web:latest/${{ matrix.app.web_image }}:${WEB_TAG}/g" $FILE
           id | grep "(root)" || sudo usermod -G root $(whoami)
           podman-compose -f $FILE up -d
-          sleep 30
+          podman exec compose_pulp_api_1 /usr/bin/wait_on_database_migrations.sh
```

Comment:
Is this what make it work with podman-compose? I'd love to hear the reasoning.

Reply:
Nope. Here the thing is to wait for all the migrations to be applied. My suspicious about the way it was working before it's because we didn't used the `healthy` checker, and within 30 seconds you have the exact amount of migrations needed to make the `status` endpoint work. It would take at least 3 or 4 more minutes to finish the migrations, which are now needed to change the status of the `api` and `content` containers to `healthy`.

Reply:
3 to 4 minutes? We _need_ to talk about our upgrade strategy once more and how we can safely squash migrations.